core/app/models/spree/shipment.rb

```diff
@@ -326,7 +326,7 @@ def update_amounts
 
     # Update Shipment and make sure Order states follow the shipment changes
     def update_attributes_and_order(params = {})
-      if update_attributes params
+      if update params
```

Comment:
what about taking advantage of the change to add parenthesis here?

Reply:
@kennyadsl done. :)